x/group/client/testutil/tx.go

```diff
@@ -112,7 +114,6 @@ func (s *IntegrationTestSuite) SetupSuite() {
 
 	s.group = &group.GroupInfo{Id: 1, Admin: val.Address.String(), Metadata: validMetadata, TotalWeight: "3", Version: 1}
 
-	// create 5 group policies
```

Comment:
is it possible below to call the function `s.createGroupThresholdPolicyWithBalance`?

In any case, let's keep this comment

Reply:
yes I can add it